### PR TITLE
Create law and test: the energy of combustion of matter

### PR DESCRIPTION
**Pull request by Voldoned**

### CODE REVIEW

File: symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from sympy import (Eq, solve)
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py:1]: Rename law to `energy_from_combustion`

File: symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(specific_heat_combustion_=specific_heat_combustion,
+    mass_of_matter_=mass_of_matter)
+@validate_output(amount_energy)
+def calculate_amount_energy(specific_heat_combustion_: Quantity, mass_of_matter_: Quantity) -> Quantity:
+
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py:28]: Remove empty line.

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py (hunk 1 above)
Comment (Voldoned) [reply symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py:1]: Do you mean rename this file?

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/the_energy_of_combustion_of_matter.py:1]: Yes

**Review by blackyblack — APPROVED** [review]

**Pull request merged**